src/main/java/com/epam/emotionalhelp/exception/EmotionNotFoundException.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.emotionalhelp.exception;
+
+public class EmotionNotFoundException extends ResourceNotFoundException {
```

Comment:
Can we create only one `ResourceNotFoundException` with `resourceId` and `resourceName` fields?

So, you can throw that exception for any model - and there is no need to create new exception class for each entity.

As for exception handler, you can use `exception.getResourceId()` to 
Can we add some new fields like `resourceId` and `resourceName` to create a response message.